.ci/docker/requirements-ci.txt

```diff
@@ -248,7 +248,7 @@ tb-nightly==2.13.0a20230426
 #Pinned versions:
 #test that import:
 
-#typing-extensions
+typing-extensions
```

Comment:
Why are we making this change?

Reply:
this is required by pytorch builds on aarch64. I'm surprised why it's not required on x86.

Reply:
Do you mind submitting typing-extensions change separately (or give me a few min to search the reason behind the change)

Reply:
I can submit separately. 
pytorch/requirement.txt lists this one:
https://github.com/pytorch/pytorch/blob/main/requirements.txt#L11

Reply:
Hi @malfet ,  This is the [PR](https://github.com/pytorch/pytorch/pull/121136) I raised for "typing_extensions", also added why it is required.

Reply:
the typing extensions PR is merged.